src/spf-explainer/i18n/en/data/long_descriptions.ts

```diff
@@ -0,0 +1,25 @@
+/*
```

Comment:
i18n files don't have copyright headers.
We can do this for all i18n files in separate PR later.